stdlib/nodejs/io.rb

```diff
@@ -43,7 +43,6 @@ def self.binread(path)
 end
 
 STDOUT.write_proc = ->(string) { `process.stdout.write(string)` }
-STDERR.write_proc = ->(string) { `process.stderr.write(string)` }
```

Comment:
Why this removal?

Reply:
I'm dumb, I looked at it multiple times and couldn't see what was different between the two lines! 🤦‍♂️